content/en/docs/examples/microservices-istio/single/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Run a Microservice Locally
+overview: Learn how to work on a single service on your local machine.
+weight: 10
+---
+
+{{< boilerplate work-in-progress >}}
+
+Before the advent of microservice architecture, development teams built,
+deployed and ran the whole application as one large chunk of software. To test a
+small change in their module not merely by unit testing, the developers had to
+build the whole application. Therefore the builds took large amount of time.
+After the build, the developers deployed their version of the application into a
+test server. The server either ran on a remote machine, or on the local computer
+where developers had to install and operate a rather complex environment on
```

Comment:
on the local computer appears twice in the sentence: _***on the local computer*** where developers had to install and operate a rather complex environment ***on their local computer***_. (Probably from my original PR, my fault).

So maybe a better sentence would be: _The developers ran the server either on a remote machine, or on their local computer. In the latter case, the developers had to install and operate a rather complex environment on their local computer._